app/assets/stylesheets/content/work_packages/timelines/elements/_bar.sass

```diff
@@ -5,9 +5,9 @@
   float: left
   z-index: 0
 
-  .timeline-element--bg
-    width: 100%
-    height: 100%
+.timeline-element--bg
+  width: 100%
+  height: 100%
 
   &:hover:not(.-clamp-style)
     .leftHandle, .rightHandle
```

Comment:
This rule is never applied as the handles are not children of the `timeline-element--bg`, but siblings.